transforms/code/license_check/ray/README.md

```diff
@@ -0,0 +1,48 @@
+# Code Quality 
+
+Please see the set of
+[transform project conventions](../../../README.md)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary
+
+This project wraps the [licence check transform](../python/README.md) with a Ray runtime.
+
+## Running
+
+### Launcher Command Line Options 
+
+
+When running the transform with the TransformLauncher, the following command line arguments are available in addition to the options provided by the python launcher.
```

Comment:
For these cli parameters you should point to the python README like others, to avoid duplication.